deploy-service/common/src/main/java/com/pinterest/deployservice/alerts/AlertAction.java

```diff
@@ -0,0 +1,16 @@
+package com.pinterest.deployservice.alerts;
+
+import com.pinterest.deployservice.bean.DeployBean;
+import com.pinterest.deployservice.bean.EnvironBean;
+
+
+/**
+ * AlertAction is the base class for an action for an alert
+ */
+public abstract class AlertAction {
+  public abstract Object perform(EnvironBean environ,
+                                 DeployBean lastDeploy,
+                                 int actionWindowInSeconds,
+                                 AlertContext context,
```

Comment:
Nit: can we put context first? I find it easier to have the more generic arguments first

Reply:
Done